apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -258,8 +258,6 @@ test.describe("Manage Booking Questions", () => {
 
       await test.step("Go to First Team Event", async () => {
         await page.getByTestId(`horizontal-tab-${team?.name}`).click();
-        await page.waitForLoadState("networkidle");
```

Comment:
`await page.waitForLoadState("networkidle");` is likely to never resolve in App Router because there are constantly likely to be network activities in App Router (e.g., streaming server components)